schemas/oic.sec.didtype.json

```diff
@@ -16,18 +14,5 @@
           },
           "required": ["uuid"]
```

Comment:
Got it.  Looking at the before and after, nothing changed so it should be the same for OCF 1.0 as it was for OIC v1.1.  Maybe the braces (i.e. JSON object) are required since it's referencing a definition in another schema file (oic.types-schema.json).  I think you and I both suffer from not fully understanding the JSON/RAML syntax.  Let's pray that things are correct.  If not, the CTT is there to keep us honest. ;)